packages/main/src/DatePicker.js

```diff
@@ -397,6 +401,25 @@ class DatePicker extends UI5Element {
 		return this._oDateFormat;
 	}
 
+	get accInfo() {
+		return {
+			"ariaDescribedBy": `${this._id}-date`,
+			"ariaHasPopup": "true",
+			"ariaAutoComplete": "none",
+			"roleAttribute": "combobox",
+			"ariaOwns": `${this._id}-popover`,
+			"ariaExpanded": this.isOpen(),
```

Comment:
What happens when the value of this.isOpen() is changed? Does something invalidate so that the component can be invalidated?

Reply:
When the picker gets opened or closed `this. _isPickerOpen` is mutated, which invalidates the `ui5-datepicker`, and  `this.isOpen()` will return either `true`, or `false` when calculated for the HBS template upon rendering. (If that was the question)